services/thermal_mgr/thermal_mgr.h

```diff
@@ -5,6 +5,7 @@
 
 typedef enum {
   THERMAL_MGR_EVENT_MEASURE_TEMP_CMD,
+  THERMAL_MGR_INTERRUPT_EVENT,
```

Comment:
nit: to match the naming convention of this enum, this should be `THERMAL_MGR_EVENT_INTERRUPT`